flask_app/config/mysqlconnection.py

```diff
@@ -8,7 +8,7 @@ def __init__(self, db):
         # change the user and password as needed
         connection = pymysql.connect(host='localhost',
                                     user='root',
-                                    password='rootroot',
+                                    password='root',
```

Comment:
I'm going to delete this from your merge and add a gitignore for future pushes that way folks can do whatever with their local copy of this file and not change the repo copy.